reaper.go

```diff
@@ -34,9 +37,22 @@ const (
 var (
 	// Deprecated: it has been replaced by an internal value
 	ReaperDefaultImage = config.ReaperDefaultImage
-	reaperInstance     *Reaper // We would like to create reaper only once
-	reaperMutex        sync.Mutex
-	reaperOnce         sync.Once
+
+	// reaperPort is the port that the reaper listens on.
+	reaperPort = nat.Port("8080/tcp")
```

Comment:
For this refactor, I'd make this port configurable with the `RYUK_PORT` env var. See moby-ryuk docs: https://github.com/testcontainers/moby-ryuk?tab=readme-ov-file#ryuk-configuration